srcs/cgi/Cgi.cpp

```diff
@@ -228,6 +229,7 @@ void Cgi::Execute() {
   if (WEXITSTATUS(status) != 0) {
     std::cerr << "The child process exited with an error" << std::endl;
     close(child_socket);
+    close(parent_socket);

```

Comment:
This is okay, right? since CGI destructor doesn't close the parent fd, then we should do it here?